views/default/view.php

```diff
@@ -152,6 +152,7 @@
     });
     
     inputField.on('fileunlock', function(event, data, previewId, index) {
+        inputField.fileinput('upload');
```

Comment:
The fileunlock event is triggered when the upload process is completed (successfully or with error). Why do we need to call upload?

Reply:
It does not upload at all.
All that happens on my side is it reloads - without a POST.
When I call the upload function - it uploads with no errors.


According to the documentation - this is only triggered after an upload - I understood this wrong. However - its not uploading at all.
https://plugins.krajee.com/file-input/plugin-events#fileunlock

Please allow me a few minutes to fix this.

Reply:
Also - the preloader just loads into infinity.
If I use JQuery - it works like a charm.